quickbooks-ruby.gemspec

```diff
@@ -26,4 +26,5 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'rspec'
   gem.add_development_dependency "webmock"
   gem.add_development_dependency 'dotenv'
+  gem.add_development_dependency 'net-http-persistent'
```

Comment:
Generally we want to have as few dependencies as possible. I recommend leaving this off and not creating a new dependency.